introduction-editor.html

```diff
@@ -53,10 +53,10 @@
 
 <h1>Using Gregorio in <span class="tex">T<span class="epsilon">e</span>X</span>Shop</h1>
 
-<p>This page describes how Gregorio is used with a <span class="tex">T<span class="epsilon">e</span>X</span> editor such as <span class="tex">T<span class="epsilon">e</span>X</span>works or <span class="tex">T<span class="epsilon">e</span>X</span>Shop. If you are used to <span class="tex">T<span class="epsilon">e</span>X</span> compilation and know the command line tools, you can  refer to the <a href="./introduction-commandline.html">command line introduction</a>.</p>
+<p>This page describes how Gregorio is used with a <span class="tex">T<span class="epsilon">e</span>X</span> editor such as <span class="tex">T<span class="epsilon">e</span>X</span>works or <span class="tex">T<span class="epsilon">e</span>X</span>Shop or VSCode. If you are used to <span class="tex">T<span class="epsilon">e</span>X</span> compilation and know the command line tools, you can  refer to the <a href="./introduction-commandline.html">command line introduction</a>.</p>
 
 
-<p>The editor must be correctly configured according to the instructions for <a href="./configuration-texworks.html"><span class="tex">T<span class="epsilon">e</span>X</span>works</a> or <a href="./configuration-texshop.html"><span class="tex">T<span class="epsilon">e</span>X</span>Shop</a>. These instructions can be easily adapted for other tools using <span class="tex">T<span class="epsilon">e</span>X</span>Shop markup, such as the <a href="https://github.com/SublimeText/LaTeXTools"><span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> Tools for SublimeText</a>.</p>
+<p>The editor must be correctly configured according to the instructions for <a href="./configuration-texworks.html"><span class="tex">T<span class="epsilon">e</span>X</span>works</a> or <a href="./configuration-texshop.html"><span class="tex">T<span class="epsilon">e</span>X</span>Shop</a> or <a href="./configuration-vscode.html" >VSCode</a>. These instructions can be easily adapted for other tools using <span class="tex">T<span class="epsilon">e</span>X</span>Shop markup, such as the <a href="https://github.com/SublimeText/LaTeXTools"><span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> Tools for SublimeText</a>.</p>
```

Comment:
With the addition of a new editor, this paragraph starts feeling cumbersome to me as it is currently formatted.  Perhaps we should consider changing the format to include a bulleted list of the links to configuration instructions?

Reply:
sure